Core/include/Acts/Vertexing/GaussianTrackDensity.hpp

```diff
@@ -116,7 +124,7 @@ class GaussianTrackDensity {
   ///
   /// @return Pair of position of global maximum and Gaussian width
   Result<std::optional<std::pair<double, double>>> globalMaximumWithWidth(
-      State& state, const std::vector<InputTrack>& trackList) const;
+      State& state, const std::vector<InputTrack>& trackList,double time) const; //do we need time here?
```

Comment:
ad comment: I don't think `time` is necessary here. But we will need to change the return type to capture `z` and `t`. Similar to this https://github.com/acts-project/acts/blob/00d84eefbb66ee9d63865e7beb72aaf22689d622/Core/include/Acts/Vertexing/AdaptiveGridTrackDensity.hpp#L102-L111